Objects/dictobject.c

```diff
@@ -739,9 +775,15 @@ new_values(size_t size)
 }
 
 static inline void
-free_values(PyDictValues *values)
+free_values(PyDictValues *values, int use_qsbr)
```

Comment:
Now that we're OK with using C99, we might as well use `bool`

```suggestion
free_values(PyDictValues *values, bool use_qsbr)
```